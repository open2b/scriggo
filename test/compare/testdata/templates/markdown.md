{# render #}

{{ "# heading" }}

heading
{{ "---" }}

heading
{{ "===" }}

{{ " text text " }}
{{ "\ttext\ttext\t" }}

{{ "**bold**" }}
{{ "__bold__" }}

{{ "*italic*" }}

{{ "***bold and italic***" }}

{{ "> blockquotes" }}
{{ ">> nested blockquotes" }}

{{ "> #### blockquote with heading" }}

{{ "1. ordered list" }}
{{ "    1. indented ordered list" }}

{{ "- unordered list" }}
{{ "+ unordered list" }}
{{ "* unordered list" }}

{{ "![image](/assets/images/image.png)" }}

{{ "`code`" }}
{{ "``code``" }}
{{ "```code```" }}

{{ "***" }}
{{ "---" }}
{{ "___" }}

{{ "[markdown](https://en.wikipedia.org/wiki/Markdown)" }}
{{ "[markdown](https://en.wikipedia.org/wiki/Markdown \"title\")" }}

{{ "<https://en.wikipedia.org/wiki/Markdown>" }}
{{ "<name@example.com>" }}

{{ "[`code`](#code)" }}

{{ "[![markdown image](/assets/images/image.jpg \"markdown\")](https://en.wikipedia.org/wiki/Markdown)" }}

{{ "\\*" }}

{{ "<em>word</em>" }}

{{ "|   |   |   |   |   |" }}
{{ "|---|---|---|---|---|" }}
