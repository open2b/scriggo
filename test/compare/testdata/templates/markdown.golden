
\# heading

heading
\-\-\-

heading
\=\=\=

 text text 
 text	text 

\*\*bold\*\*
\_\_bold\_\_

\*italic\*

\*\*\*bold and italic\*\*\*

\> blockquotes
\>\> nested blockquotes

\> \#\#\#\# blockquote with heading

1\. ordered list
    1\. indented ordered list

\- unordered list
\+ unordered list
\* unordered list

\!\[image\]\(/assets/images/image\.png\)

\`code\`
\`\`code\`\`
\`\`\`code\`\`\`

    code block
 code block

    a
    ***b***
	a
	***b***

\*\*\*
\-\-\-
\_\_\_

\[markdown\]\(https://en\.wikipedia\.org/wiki/Markdown\)
\[markdown\]\(https://en\.wikipedia\.org/wiki/Markdown "title"\)

\<https://en\.wikipedia\.org/wiki/Markdown\>
\<name@example\.com\>

\[\`code\`\]\(\#code\)

\[\!\[markdown image\]\(/assets/images/image\.jpg "markdown"\)\]\(https://en\.wikipedia\.org/wiki/Markdown\)

\\\*

\<em\>word\</em\>

\|   \|   \|   \|   \|   \|
\|\-\-\-\|\-\-\-\|\-\-\-\|\-\-\-\|\-\-\-\|
